integ/main_test.go

```diff
@@ -36,8 +36,8 @@ func (s *IntegSuite) SetUpSuite(c *C) {
 	}
 	// var runDirect = flag.Bool("runDirect", false, "if set, runs the operator from the current source instead of from an image")
 	var runDirect = os.Getenv("RUN_DIRECT") != ""
-	// var image = flag.String("image", "lyft/flinkk8soperator:latest", "image for the operator")
-	var image = os.Getenv("OPERATOR_IMAGE")
+	// var image = flag.String("image", "flinkk8soperator:latest", "image for the operator")
+	var image = os.Getenv("IMAGE")
```

Comment:
I think OPERATOR_IMAGE is clearer, as there are several different images involved.

Reply:
Yes. The documentation was IMage. Will change to Operator_image everywhere

Reply:
Yeah, I forgot to update the doc when I changed it a while back.